free/src/test/scala/cats/free/YonedaTests.scala

```diff
@@ -0,0 +1,30 @@
+package cats
+package free
+
+import cats.tests.CatsSuite
+import cats.laws.discipline.{ArbitraryK, FunctorTests, SerializableTests}
+
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop.forAll
+
+class YonedaTests extends CatsSuite {
+  implicit def yonedaArbitraryK[F[_] : Functor](implicit F: ArbitraryK[F]): ArbitraryK[Yoneda[F, ?]] =
+    new ArbitraryK[Yoneda[F, ?]]{
+      def synthesize[A: Arbitrary]: Arbitrary[Yoneda[F, A]] =
+        Arbitrary(F.synthesize[A].arbitrary.map(Yoneda(_)))
+    }
+
+  implicit def yonedaArbitrary[F[_] : Functor : ArbitraryK, A : Arbitrary]: Arbitrary[Yoneda[F, A]] = yonedaArbitraryK[F].synthesize[A]
```

Comment:
So far I think we have been putting Arbitrary and ArbitraryK instances in the `Arbitrary.scala` and `ArbitraryK.scala` files. Do we want these there as well?